ayon_server/events/eventstream.py

```diff
@@ -15,25 +15,44 @@
 
 class EventStream:
     model: Type[EventModel] = EventModel
-    hooks: dict[str, dict[str, HandlerType]] = {}
+    local_hooks: dict[str, dict[str, HandlerType]] = {}
+    global_hooks: dict[str, dict[str, HandlerType]] = {}
```

Comment:
With this you can get rid of multiple `if topic not in *hooks*` conditions and avoid possible missing keys issues.
```suggestion
    local_hooks: dict[str, dict[str, HandlerType]] = collections.defaultdict(dict)
    global_hooks: dict[str, dict[str, HandlerType]] = collections.defaultdict(dict)
```